src/components/vms/hostvmslist.tsx

```diff
@@ -82,6 +87,41 @@ const VmState = ({
     );
 };
 
+const VmUsageCpu = ({ vm } : {vm: VM | UIVM, }) => {
+    if (vm.isUi || vm.state != "running")
+        return null;
+
+    const vmCpuUsage: number = vm.cpuUsage ? vm.cpuUsage : 0;
+    const cpuUsage = vm.cpuUsage && isNaN(vmCpuUsage) ? 0 : parseFloat(vmCpuUsage.toFixed(1));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2183-bb984df6-20250908-152215-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)